.eslintrc.json

```diff
@@ -14,8 +19,17 @@
         "array-callback-return": "error",
         "array-element-newline": "off",
         "arrow-body-style": "error",
-        "arrow-parens": "error",
-        "arrow-spacing": "error",
+        "arrow-parens": [
+            "error",
+            "as-needed"
+        ],
+        "arrow-spacing": [
+            "error",
+            {
+                "after": true,
+                "before": true
+            }
```

Comment:
these options are the default of `arrow-spacing`. what’s the reason to list them out?

Reply:
Because they come in conflict with prettier I think, I can play with both configs and see if I can roll this back. Honestly, handling both linter and formatting at the same time it's kind of a pain.

Reply:
I would just turn off or delete every stylistic eslint rule and let prettier manage it exclusively.

Reply:
Interesting, I like that eslint prevents some _doodoo_ on the code, but will definitely give it a try locally and see how it works when something breaks.